site/mkdocs.yml

```diff
@@ -29,7 +29,10 @@ nav:
 - Reference:
   - reference/crds.md
   - Connectors API: reference/connectors.md
-  - Components: 
+  - Connectors API (OpenAPI): 
+    - DataCatalog: reference/openapigenerator-doc-datacatalog/README.md
+    - PolicyManager: reference/openapigenerator-doc-policymanager/README.md
```

Comment:
I suggest to call these folders `connectors-datacatalog` and `connectors-policymanager`.
Also the titles should be `Data catalog` and `Policy manager`.

Reply:
fixed. Please check.